site/content/docs/5.0/examples/features/index.html

```diff
@@ -6,64 +6,6 @@
 body_class: ""
 ---
 
-<svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
Hum so you did remove them. However they're probably called somewhere with the `<use>` tag and `symbol`s' `id`, aren't they?

Reply:
yes ;-)